src/task/style.css

```diff
@@ -258,3 +258,23 @@ footer {
   color: rgb(255, 92, 92);
   font-size: 0.75rem;
 }
+
+#assigneeAvatar {
+  background: right / 42px;
+  position: absolute;
+  top: 33%;
+  width: 40px;
+  height: 40px;
+  right: 3%;
+  border-radius: 50%;
+  display: none;
```

Comment:
A better approach would be to move the `display:none` to a separate reusable class and keep the style of the avtaar as it is.